tests/src/Functional/PersonContentRenderTest.php

```diff
@@ -0,0 +1,296 @@
+      $article->filter('h1.card-title')->text()
+    );
+    $image = $article->filter('img.card-img-top');
+    $this->assertCount(1, $image);
+    $this->assertStringContainsString(
+      'image-test.png',
+      $image->attr('src')
+    );
+    $this->assertEquals(
+      'DG Test',
+      $article->filter('.card-body > div.my-3 > span:nth-child(1)')->text()
+    );
+    $this->assertEquals(
+      'Director',
+      $article->filter('.card-body > div.my-3 > span:nth-child(2)')->text()
+    );
+  }
+
+  /**
+   * Creates an example person node only with required fields.
+   *
+   * @return \Drupal\node\NodeInterface
+   *   Person node.
+   */
+  protected function createExamplePersonWithRequiredFieldsOnly(): NodeInterface {
+    // Create a Person node with required fields.
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = \Drupal::entityTypeManager()
+      ->getStorage('node')
+      ->create([
+        'type' => 'oe_sc_person',
+        'oe_sc_person_first_name' => 'Stefan',
+        'oe_sc_person_last_name' => 'Mayer',
+        'oe_sc_person_country' => 'DE',
+        'oe_sc_person_occupation' => 'DG Test',
+        'oe_sc_person_position' => 'Director',
+        'uid' => 1,
+        'status' => 1,
+      ]);
+    $node->save();
+    return $node;
+  }
+
+  /**
+   * Creates an example person node with required and optional fields.
+   *
+   * @return \Drupal\node\NodeInterface
+   *   Person node.
+   */
+  protected function createExamplePersonWithAllFields(): NodeInterface {
+    /** @var \Drupal\node\Entity\Node $node */
+    $node = $this->createExamplePersonWithRequiredFieldsOnly();
+    // Create a sample image media entity to be embedded.
+    $image_file = File::create([
+      'uri' => $this->getTestFiles('image')[0]->uri,
+    ]);
+    $image_file->save();
+    $media_image = Media::create([
+      'bundle' => 'image',
+      'name' => 'Starter Image test',
+      'oe_media_image' => [
+        [
+          'target_id' => $image_file->id(),
+          'alt' => 'Starter Image test alt',
+          'title' => 'Starter Image test title',
+        ],
+      ],
+    ]);
+    $media_image->save();
+
+    // Create a sample document media entity to be embedded.
+    $document_file = File::create([
+      'uri' => $this->getTestFiles('text')[0]->uri,
+    ]);
+    $document_file->save();
+    $media_document = Media::create([
+      'bundle' => 'document',
+      'name' => 'Person document test',
+      'oe_media_file_type' => 'local',
+      'oe_media_file' => [
+        [
+          'target_id' => (int) $document_file->id(),
+          'alt' => 'Person document alt',
+          'title' => 'Person document title',
+        ],
+      ],
+    ]);
+    $media_document->save();
+
+    $document_reference = \Drupal::entityTypeManager()
+      ->getStorage('oe_document_reference')
+      ->create([
+        'type' => 'oe_document',
+        'oe_document' => $media_document,
+        'status' => 1,
+      ],
+      );
```

Comment:
```suggestion
      ]);
```
The opening `([` are on the same line, so the closing `])` should be too.